sentinel5dl/__init__.py

```diff
@@ -21,6 +22,21 @@
 PASS = 's5pguest'
 
 
+def __md5(filename):
+    """Generate the md5 sum of a file
+
+    :param filename: input filename for which the md5 sum is generated.
+    :type filename: str
+    :return: hex representation of the md5 sum with upper letters.
+    :rtype: str
+    """
+    hash_md5 = hashlib.md5()
+    with open(filename, "rb") as f:
```

Comment:
Here and at other places in this pull request, can we go with `'` for consistency?

```suggestion
    with open(filename, 'rb') as f:
```

Reply:
I also tried to remove all other `"` I introduces :smile: